src/GB2260.php

```diff
@@ -16,37 +16,41 @@ public static function getData()
     public static function parse($code)
     {
         if (empty(self::$_data)) {
-            self::$_data = require 'data.php';
+            self::$_data = require dirname(__FILE__) . DIRECTORY_SEPARATOR . 'data.php';
         }
 
+        $code = (string)$code;
         $code = preg_replace('/0+$/', '', $code);
         $codeLength = strlen($code);
         if ($codeLength < 2 || $codeLength > 6 || $codeLength % 2 !== 0) {
-            throw new \Exception('Invalid code');
+            throw new \InvalidArgumentException('Invalid code');
         }
 
-        $province = self::$_data[substr($code, 0, 2) . '0000'];
-        if (!$province) {
+        $provinceKey = intval(substr($code, 0, 2) . '0000');
+        if(!array_key_exists($provinceKey, self::$_data)){
```

Comment:
Missing required space, use

```php
if (!array_key_exists($provinceKey, self::$_data)) {
```

instead of

```php
if(!array_key_exists($provinceKey, self::$_data)){
```